connectors/src/connectors/slack/init.ts

```diff
@@ -0,0 +1,56 @@
+import { Connector, SlackConfiguration, sequelize_conn } from '@app/lib/models';
+import { Err, Ok, Result } from '@app/lib/result';
+import { Nango } from '@nangohq/node';
+import { WebClient } from '@slack/web-api';
+import { DustConfig } from '@app/types/dust_config';
+export type NangoConnectionId = string;
+
+const { NANGO_SECRET_KEY, NANGO_SLACK_CONNECTOR_ID } = process.env;
+
+export async function createSlackConnector(
```

Comment:
The interface to front should be a generic function that takes a `connectorType = "slack" | ...`

Reply:
Actually not sure about this one... it's kind of equivalent so following your lead on this one :+1:

Reply:
I did it that way thinking the interface could be different for each connector. So i will keep it the way it is.

Reply:
:+1: